kernel/thread.c

```diff
@@ -996,6 +1001,11 @@ static long _run_thread(void* arg_)
             {
                 long r = myst_munmap(
                     thread->main.exec_crt_data, thread->main.exec_crt_size);
+                if (myst_mman_pids_set(
```

Comment:
`myst_mman_pids_set` with process id `0` follows `myst_munmap` in this PR.
Can `myst_munmap` do `myst_mman_pids_set` instead of the caller having to do it?